recipes/tableauhyperapi/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "tableauhyperapi" %}
+{% set version = "0.0.11556" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-manylinux2014_x86_64.whl 
```

Comment:
conda-forge does not build from `.whl` files. You need a source distribution. See general comment